### PR TITLE
[Feature] Include OpenEBS Addon

### LINKED ISSUE #2079: Feature - OpenEBS Addon
This is a feature request to have a storage which works better with a multi node MicroK8s.

### PR DESCRIPTION
**Pull request by balchua**
### Include OpenEBS to the list of MicroK8s addon

Why OpenEBS?  Its just easy to install, less friction. It supports local storage (for those on single node) as well as iscsi based.

Requires that all nodes have `iSCSI` client installed.  

More information on OpenEBS [here](https://docs.openebs.io/)

Closes #2079

This will potentially address several issues related to storage with a multi node MicroK8s setup.

*Also verify you have:*
* [X] Read the [contributions](https://github.com/ubuntu/microk8s/blob/master/CONTRIBUTING.md) page.
* [X] Submitted the [CLA form](https://ubuntu.com/legal/contributors/agreement), if you are a first time contributor.

### CODE REVIEW

File: microk8s-resources/actions/enable.openebs.sh [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+# make sure the "openebs" namespace exist
+$KUBECTL create namespace "$OPENEBS_NS" > /dev/null 2>&1 || true
+
+HELM="$SNAP_DATA/bin/helm3 --kubeconfig=$SNAP_DATA/credentials/client.config"
+
+$HELM repo add openebs https://openebs.github.io/charts
+$HELM repo update
+$HELM -n openebs install openebs openebs/openebs \
+    --set varDirectoryPath.baseDir="$SNAP_COMMON/var/openebs/" \
+    --set jiva.defaultStoragePath="$SNAP_COMMON/var/openebs/"
```
Comment (tjmchenry) [inline microk8s-resources/actions/enable.openebs.sh:34]: I noticed the ndm sparse path wasn't set to inside the snap so I looked through the doc I think this is all the /var/openebs directories that need setting. I added them and removed trailing slashes from the existing two.
reference:
https://github.com/openebs/charts/blob/master/charts/openebs/README.md

```suggestion
    --set varDirectoryPath.baseDir="$SNAP_COMMON/var/openebs" \
    --set jiva.defaultStoragePath="$SNAP_COMMON/var/openebs" \
    --set localprovisioner.basePath="$SNAP_COMMON/var/openebs/local" \
    --set ndm.sparse.path="$SNAP_COMMON/var/openebs/sparse"
```

Comment (balchua) [reply microk8s-resources/actions/enable.openebs.sh:34]: Thanks for the review.  Good catch.  I will update the PR. 👍

**Review by ktsakalozos — APPROVED** [review]
LGTM +1 great work @balchua

**Comment by ktsakalozos** [comment]
Thank you for this work @balchua and @tjmchenry for the review.

**Pull request merged**